Binoculars.csproj

```diff
@@ -148,21 +148,23 @@
       <SourcePkg Include="$(ProjectDir)credentials.json" />
       <SourceExtension Include="$(TargetDir)*ExtensionDefinition.xml" />
     </ItemGroup>
+    <!--Delete any previous build folder -->
+    <RemoveDir Directories="$(TargetDir)$(ProjectName)" />
```

Comment:
I've added a RemoveDir. Let me know if this is safe or could potentially break things?